src/dotnet-scaffolding/dotnet-scaffold-aspnet/Commands/AreaCommand.cs

```diff
@@ -20,20 +20,20 @@ public AreaCommand(IFileSystem fileSystem, ILogger logger, IEnvironmentService e
         _logger = logger;
     }
 
-    public override int Execute([NotNull] CommandContext context, [NotNull] AreaSettings settings)
+    public Task<int> ExecuteAsync(AreaCommandSettings settings)
     {
         if (!ValidateAreaCommandSettings(settings))
         {
-            return -1;
+            return Task.FromResult(-1);
```

Comment:
I'm 90% kidding, but we should find some trivial thing that is an actual async call that most things will call so everyone doesn't have to do the Task.FromResult stuff if they don't do any async work on their own 🤣

Reply:
I know spectre has a way of instantiating async and non-async commands and eventually we would do something similar.